data/sles4sap/qe_sap_deployment/mr_test_gcp.yaml

```diff
@@ -11,6 +11,7 @@ terraform:
     monitoring_os_major_version: "%HANA_OS_MAJOR_VERSION%"
     drdb_os_major_version: "%HANA_OS_MAJOR_VERSION%"
     netweaver_os_major_version: "%HANA_OS_MAJOR_VERSION%"
+    bastion_os_major_version: "%HANA_OS_MAJOR_VERSION%"
```

Comment:
Why is it needed?

Also take care that inspecting one of your [verification](https://openqa.suse.de/tests/10634612#settings) I noticed that settings are like

```
QESAPDEPLOY_GITHUB_BRANCH  main
QESAPDEPLOY_GITHUB_REPO  github.com/SUSE/qe-sap-deployment
QESAP_INSTALL_VERSION  0.6.0
```
Practically speaking, due to how code is organized in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/48270b1b56f6f0dc5ad02121e24b726af8601404/lib/qesapdeployment.pm#L209 (for the moment), QESAP_INSTALL_VERSION win and QESAPDEPLOY_GITHUB_BRANCH+QESAPDEPLOY_GITHUB_REPO are ignored. In any case, for this kind of test, I suggest to use QESAP_INSTALL_VERSION. For clearness I suggest to remove QESAPDEPLOY_GITHUB_BRANCH+QESAPDEPLOY_GITHUB_REPO.

So for version 0.6.0 of the qe-sap-deployment, bastion_os_major_version is still valid, but it will be soon removed https://github.com/SUSE/qe-sap-deployment/pull/128/files#diff-7984cbbcf2550ec2c4dd549fcabf8170964f4b33863f9cc1298a89e0eb70e41eL79  and https://github.com/SUSE/qe-sap-deployment/pull/128/files#diff-39bb739d46e970e42ffec0913c80415c56893dca6666056b842ebc2545fe99b9L861

Reply:
> I do not see bastion variable in https://openqa.suse.de/tests/10634612/file/qesap_terraform-mr_test_gcp.yaml

I can find `bastion_os_major_version: '15'` in this link.